openwisp_controller/config/admin.py

```diff
@@ -1362,6 +1362,14 @@ class ConfigSettingsInline(admin.StackedInline):
         model = OrganizationConfigSettings
         form = ConfigSettingsForm
 
+        def get_fields(self, request, obj=None):
+            fields = ["context"]
```

Comment:
Doesn't this change the order in the UI? The config variables field should be last.

Reply:
Yes it does change, i will retain the original order.